src/core/cgroup.c

```diff
@@ -189,6 +189,7 @@ void cgroup_context_dump(CGroupContext *c, FILE* f, const char *prefix) {
                 "%sCPUShares=%" PRIu64 "\n"
                 "%sStartupCPUShares=%" PRIu64 "\n"
                 "%sCPUQuotaPerSecSec=%s\n"
+                "%sCPUQuotaPeriodUSec=%s\n"
```

Comment:
USec → Sec

Reply:
So here I was trying to keep the DBus names, which typically use `USec` suffix on settings that use `Sec` in the config file...

Though I see the one for `CPUQuota` doesn't really match exactly the one from DBus, since it's `PerSecUSec` there and `PerSecSec` here... What would be correct here do you think?

Should I adjust the `CPUQuotaPerSecSec=` here to become `CPUQuotaPerSecUSec=`? (and stop using format_timespan there) or should I use `CPUQuotaPeriodSec=` here? (If the former, then why not report `CPUQuota=` with a percentage value here, since that would match the actual configuration?)

Reply:
we try to keep the prop names in unit files and on the bus objects in sync, but the former usually leans more to the "user friendly" side, while the latter to the "normalized" side. This means the unit file usually lists stuff with seconds as default unit, while the latter exposes stuff normalized to usec like we use everyhere else. 

the "dump" calls exist only for debugging, hence we are a bit sloppy there, but usually we show the user friendly version, except where we don't. If in doubt we should probably show the user friendly version there, unless there's a good reason to show the low-level version...